### PR TITLE
DOCS Implement canView checks into ReadMembersQueryCreator examples

### PR DESCRIPTION
**Pull request by robbieaverill**
Recreates #42

### CODE REVIEW

File: examples/code/ReadMembersQueryCreator.php [hunk 1]
```diff
@@ -1,35 +1,54 @@
...
+use GraphQL\Type\Definition\ResolveInfo;
 use GraphQL\Type\Definition\Type;
 use SilverStripe\GraphQL\QueryCreator;
+use SilverStripe\GraphQL\OperationResolver;
 use SilverStripe\Security\Member;
-use SilverStripe\GraphQL\Pagination\Connection;
-use SilverStripe\GraphQL\Pagination\PaginatedQueryCreator;
-use SilverStripe\GraphQL\Manager;
 
-class ReadMembersQueryCreator extends PaginatedQueryCreator
```
Comment (chillu) [inline examples/code/ReadMembersQueryCreator.php]: The README example is now out of sync with this file - both should use `PaginatedQueryCreator`

File: README.md [hunk 2]
```diff
@@ -160,7 +160,7 @@ SilverStripe\GraphQL:
 You can query data with the following URL:
 
 ```
-/graphql?query=query+readMembers{members{ID,Email,FirstName,Surname}}
```
Comment (chillu) [inline README.md]: The endpoing example will need to be updated to work with the `PaginatedQueryCreator` syntax (there'll need `edges` and `node` references in there)

File: examples/code/ReadMembersQueryCreator.php (hunk 1 above)
Comment (robbieaverill) [inline examples/code/ReadMembersQueryCreator.php]: Cool. Should the examples all be paginated by default now?

**Comment by robbieaverill** [comment]
@chillu I've updated the readme and examples for `readMember` to both use `PaginatedQueryCreator`

File: examples/code/ReadMembersQueryCreator.php (hunk 1 above)
Comment (chillu) [inline examples/code/ReadMembersQueryCreator.php]: Yeah it makes sense right? It's a bit dangerous to have unpaginated ones, given the ORM will choke at a couple of dozen items depending on memory limit constraints.

**Comment by chillu** [comment]
Thanks Robbie :)

**Pull request merged**